### DISCUSSION TITLE
How OSPOs do metrics on public repos to share upwards

### DISCUSSION THREAD

**Discussion by anajsana** [todogroup/ospology#41] [OSPO Discussions]
Discussion Thread from [TODO Slack channel](https://slack.todogroup.org/):

**David Burns (BrowserStack)**

Hey folks, I was wondering how people do metrics on public repos to share upwards. Internally we use Jira and it's easy to extract the info as there are a number of things that need to be met in open issues. Unfortunately, we can have an open Jira board for reasons and have been using github as that's where all our open source code. We want to main the openness of our issues especially as we try build out community projects so others can help if they want.

So, My question is, how do people track velocity, workloads, etc, against their company ways when internal tracking and external tracking is different without generating a lot of work for yourself. Our main work is in an Org so currently have a spreadsheet of all the repos we worked on each month to feed this up to our CTO.

**Comment by anajsana** [comment]
@vmbrasseur : Check out https://cauldron.io and/or the other Bitergia products. They're the repo data pros

**Comment by anajsana** [comment]
@justaugustus : There's also Orbit, which may suit your needs: https://orbit.love/

Some notes on Community CRMs here: https://github.com/cncf/tag-contributor-strategy/blob/main/contributor-growth/drafts/crm-runbook.md

**Reply by jdorfman** [reply]
Orbit is amazing. I use it daily to track metrics for Curiefense. Highly recommend it to all OSPOers.
